Jellyfin.Plugin.Anime/Configuration/configPage.html

```diff
@@ -73,11 +73,11 @@
 
                             $('#titleLanguage', page).val(config.TitlePreference).change();
                             $('#chkMaxGenres', page).val(config.MaxGenres).change();
-                            $('#chkTidyGenres', page).checked = config.TidyGenreList;
-                            $('#chkAddAnimeGenre', page).checked = config.AddAnimeGenre;
+                            document.getElementById('chkTidyGenres').checked = config.TidyGenreList;
```

Comment:
```suggestion
                            $('#chkTidyGenres', page).prop('checked', config.TidyGenreList);
```
I would keep jQuery for consistency with the rest of the code

Reply:
We're currently in the process (very extended) of completely removing jQuery from the codebase, I just didn't have time to remove the rest of the usages here.

Reply:
The goal is to eventually remove the HTML requirement from most of these plugins as well, since they just expose it for basic configuration.

Reply:
Ok, I'm making some changes to the configuration page and will take this into account.